SlicerRadiomics/SlicerRadiomics.py

```diff
@@ -278,8 +295,15 @@ def onApplyButton(self):
     kwargs['verbose'] = self.verboseCheckBox.checked
     kwargs['label'] = int(self.labelSliderWidget.value)
```

Comment:
Sure, I will do that.

Arguments passing will need to be revisited, but I thought we can leave that for another PR.